forge-gui/res/cardsfolder/upcoming/mordor_on_the_march.txt

```diff
@@ -0,0 +1,12 @@
+Name:Mordor on the March
+ManaCost:3 B R
+Types:Sorcery
+K:Storm
+A:SP$ ChooseCard | Mandatory$ True | Choices$ Creature.YouCtrl  | ChoiceZone$ Graveyard | SubAbility$ DBChangeZone | RememberChosen$ True | SpellDescription$ Exile a creature card from your graveyard. Create a token that's a copy of it. It gains haste until end of turn. Exile it at the beginning of the next end step.
```

Comment:
No reason for extra ChooseCard imo

Reply:
I can't seem to get it to work with changetype

Reply:
you need Hidden$ True if you want to to use it on a public zone

Reply:
Thanks working now